contracts/src/v0.8/vrf/dev/VRFV2PlusWrapperConsumerBase.sol

```diff
@@ -119,4 +119,16 @@ abstract contract VRFV2PlusWrapperConsumerBase {
     require(msg.sender == address(VRF_V2_PLUS_WRAPPER), "only VRF V2 Plus wrapper can fulfill");
```

Comment:
Lets fix this to revert with a custom error, should reduce code size and satisfy the linter.

Reply:
Done